scripts/set-more-info-link.py

```diff
@@ -114,7 +117,10 @@ def main():
     rel_paths = []
 
     if not args.page.lower().endswith('.md'):
+        package = args.page.lower()
         args.page = f'{args.page}.md'
+    else:
+        package = args.page.lower().rstrip('.md')
```

Comment:
`package` can be declared where it's used (I'd also like it called `page_name` because that's clearer)
```suggestion
    if not args.page.lower().endswith('.md'):
        args.page = f'{args.page}.md'
```